gcode.c

```diff
@@ -89,6 +88,7 @@ static uint8_t gc_check_same_position(float *pos_a, float *pos_b)
 // coordinates, respectively.
 uint8_t gc_execute_line(char *line) 
 {
+  report_debug_message("received gcode");
```

Comment:
If this prints all the time without a compiler flag being set, it should be removed.